public/js/santaAdmin.js

```diff
@@ -73,8 +73,19 @@ $(async function() {
   $.getJSON(`api/forbidden?groupId=${groupId}`, function(result) {
     // TODO make this beautiful
     result.forEach((pair, index) => {
-      $('#forbiddenPairsTable tbody').append(`<tr><td><b>${++index}</b></td><td>${pair.forbiddenPair1}</td><td>${pair.forbiddenPair2}</td></tr>`);
+      $('#forbiddenPairsTable tbody').append(`<tr><td><b>${++index}</b></td><td class="pairColumn1">${pair.forbiddenPair1}</td><td>${pair.forbiddenPair2}</td><td><i class="buttonDelete bi bi-trash" style='color:red'></i></td></tr>`);
     });
+    $('.buttonDelete').on('click', function ()  {
+        const pair = {
```

Comment:
What's this used for?

Reply:
We use the const keyword in JavaScript to declare variables whose value can be initialized only at the time of declaration.

Reply:
What I meant was: what's this object `pair` used for? Why did you create it? What's its purpose?